src/config/db.tsx

```diff
@@ -33,11 +33,25 @@ export const documentSchema: RxJsonSchema<DocumentProperties> = {
     },
     document: {
       type: "object"
+    },
+    qrCode: {
```

Comment:
Wondering why don’t want to just tag it to the document itself?

Reply:
Also I don’t see any link between document and qrCode, but I might be missing something...